src/commands/General/User Settings/userconf.js

```diff
@@ -24,46 +28,108 @@ module.exports = class extends Command {
 	}
 
 	show(message, [key]) {
-		const piece = this.getPath(key);
-		if (!piece || (piece.type === 'Folder' ? !piece.configurableKeys.length : !piece.configurable)) return message.sendLocale('COMMAND_CONF_GET_NOEXT', [key]);
-		if (piece.type === 'Folder') {
-			return message.sendLocale('COMMAND_CONF_USER', [
-				key ? `: ${key.split('.').map(toTitleCase).join('/')}` : '',
-				codeBlock('asciidoc', message.author.settings.display(message, piece))
-			]);
+		const schemaOrEntry = this.configurableSchemaKeys.get(key);
+		if (typeof schemaOrEntry === 'undefined') throw message.language.get('COMMAND_CONF_GET_NOEXT', key);
+
+		const value = key ? message.author.settings.get(key) : message.author.settings;
+		if (schemaOrEntry.type !== 'Folder') {
+			return message.sendLocale('COMMAND_CONF_GET', [key, this.displayEntry(schemaOrEntry, message.author.settings.get(key))]);
```

Comment:
```suggestion
			return message.sendLocale('COMMAND_CONF_GET', [key, this.displayEntry(schemaOrEntry, value)]);
```

For both of these you could just use the `value` instead of getting again